packages/dd-trace/src/gateway/als.js

```diff
@@ -0,0 +1,5 @@
+'use strict'
+
+const { AsyncLocalStorage } = require('async_hooks')
+
+module.exports = new AsyncLocalStorage()
```

Comment:
Is this passthrough module necessary? It seems like it isn't.

Reply:
Ah, I misunderstood.

Still though, it looks like it's only used in engine/index.js and engine/runner.js. If you need a shared instance, maybe it can live on one of those files and be passed appropriately?

Consider this a non-blocking suggestion.